schemas/test/en/test_dynamic_answers_list.json

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
You will need 1 with static answers, 1 without. Currently, answers are expected in certain parts of runner but they haven't been updated to handle that it might no longer exist. So testing both will highlight those issues. Tests should be updated accordingly.

Reply:
I've added one without static to test schema, it seems to be working fine when running functional tests against it.